Revise contributing guidance (#915)
short-d/short: CONTRIBUTING.md

- Creating UI mockups
- Organizing or participating sync meetings

After 2 months of consistent high quality contributions, you will receive an invitation to join the core team, 
where people together plan the product roadmap for Short, design complex systems, create new libraries, building 
distributed cloud infrastructure & drive innovations.

We are specificly looking for the following characters for the potential member of the core team:

> - You are excited about learn new things and learn them very fast.

Reviewer: ```suggestion
- You are excited about learning new things and learning them very fast.
```

- You write clean code & constantly look for corners to simplify the existing systems.

> - You believe in automatiom instead of doing everything manually in the long run.

Reviewer: ```suggestion
- You believe in automation instead of doing everything manually in the long run.
```

- You strive for excellence in term of code quality, system design, and process optimization.
- You are a team player, who schedule pair programming sessions & solve difficult problems together with team members.

> - You thrieve in complexities & difficulties. Compilers, operating systems, pragramming languages, bitwise optimizations, 

Reviewer: ```suggestion
- You thrive in complexities & difficulties. Compilers, operating systems, programming languages, bitwise optimizations, 
```

> distributed systems, machine learning vector optimization, computer vision algorithms excites you.

Author: ```suggestion
distributed systems, machine learning vector optimization, computer vision algorithms excite you.
```
